domain/src/main/kotlin/no/nav/su/se/bakover/domain/beregning/Sats.kt

```diff
@@ -1,13 +1,36 @@
 package no.nav.su.se.bakover.domain.beregning
 
+import no.nav.su.se.bakover.common.periode.Periode
 import no.nav.su.se.bakover.domain.Grunnbeløp
 import java.math.BigDecimal
 import java.time.LocalDate
+import kotlin.math.roundToInt
 
 enum class Sats(val grunnbeløp: Grunnbeløp) {
     ORDINÆR(Grunnbeløp.`2,28G`),
     HØY(Grunnbeløp.`2,48G`);
 
-    fun fraDato(dato: LocalDate): Double = grunnbeløp.fraDato(dato)
-    fun fraDatoAsInt(dato: LocalDate): Int = BigDecimal.valueOf(grunnbeløp.fraDato(dato)).divide(BigDecimal.ONE, 0, java.math.RoundingMode.HALF_UP).toInt()
+    private val MÅNEDER = 12L
+    private val MÅNEDER_PRESISJON = 2
+    private val ÅR = 1L
+    private val ÅR_PRESISJON = 0
+
+    fun årsbeløp(dato: LocalDate): Int = satsSomÅrsbeløp(dato)
```

Comment:
Stemmer det at denne gir 1G for gjeldende grunnbeløp på den datoen som sendes inn?

En ting det kunne tolkes som er at denne gir totalsummen av en utbetaling lik 1G i ett år fra og med datoen som sendes inn; altså at den kan bli høyere enn 1G på `dato`, dersom det er en G-justering en gang mellom `dato` og `dato + 1 år`.

Vet ikke om det skaper noe trøbbel/forvirring nå (det gjør det forhåpentligvis ikke siden vi har tester for det), men kanskje vi skal notere oss det bak øret i tilfelle vi trenger den oppførselen.

Reply:
Det stemmer ikke. Funksjonen gir oss årsbeløpet for HØY/ORDINÆR sats regnet på innsendt dato (uten å ta hensyn til eventuelle endringer i G). Denne er for øvrig bare i bruk av den "gamle" koden (den jeg forsøker å erstatte).

Reply:
Hehe, okei. Får ha det i bakhodet når vi lager ny versjon da; om vi klarer å være enda tydeligere på dette.